Makefile

```diff
@@ -9,9 +9,9 @@ ifeq (test_one,$(firstword $(MAKECMDGOALS)))
 endif
 
 clean:
-	find -name '*.pyc' -delete
-	find -name '*~' -delete
-	find -name '__pycache__' -type d -delete
+	find . -name '*.pyc' -delete
+	find . -name '*~' -delete
+	find . -name '__pycache__' -type d -delete
```

Comment:
@Kelketek You seem to have dropped it already so you might have found it in the meantime, but the latest master has the Vagrantfile: https://github.com/open-craft/opencraft/blob/master/Vagrantfile (cf the README also).